test/electron/features/native-crash.feature

```diff
@@ -13,8 +13,15 @@ Feature: Native Errors
       | events    | 0 |
       | minidumps | 1 |
       | sessions  | 2 |
-    And minidump request 0 contains a file form field named "upload_file_minidump"
-    And minidump request 0 contains a form field named "event" matching "minidump-event.json"
+    Then minidump request 0 contains a file form field named "upload_file_minidump"
+    Then minidump request 0 contains a form field named "event" matching "minidump-event.json"
+
+  Scenario: Non-main minidumps are detected and uploaded
+    Given I launch an app
+    And I click "renderer-process-crash"
```

Comment:
nit: Starting the app and clicking are both actions caused by the step rather than existing conditions.

```suggestion
    When I launch an app
    And I click "renderer-process-crash"
```